wooey/templates/wooey/scripts/script_view.html

```diff
@@ -14,6 +14,13 @@
     <div class="page-header">
         <h3 id="wooey-script-title">{{ script.script_name }}</h3>
         <p id="wooey-script-description">{{ script.script_description }}</p>
+
+        <p id="wooey-script-info" style="color:gray"><small>
```

Comment:
This is ok, but I'll probably move it elsewhere. Seems like we're putting a lot of stuff above where users primarily interact with the script.

Reply:
Also should handle the none case better than just saying none

Reply:
Sounds good. Do you think somewhere a bit more discrete, like a footer?

Reply:
I was thinking to put the information in the header, and only put the last modified/version stuff in. like:
![image](https://user-images.githubusercontent.com/963909/183667712-1a7b8878-68c1-4f18-904c-0b0b788ffd49.png)

And if the user is unknown -- omit it.

Reply:
<img width="652" alt="image" src="https://user-images.githubusercontent.com/1281521/183678266-9e1c5e49-b86f-4882-9b3b-9eb035182f9d.png">